examples/io/plot_read_events.py

```diff
@@ -0,0 +1,77 @@
+"""
+.. _ex_read_events:
```

Comment:
I am just trying to be consistent here. It's an example and we refer to examples this way